docs/virtualkeyboard/index.html

```diff
@@ -0,0 +1,492 @@
+                  }, {
+                      value: 'Commit history.',
+                      href: 'https://github.com/w3c/editing/commits/gh-pages'
+                  }, {
+                      value: 'Mailing list.',
+                      href: 'http://lists.w3.org/Archives/Public/public-editing-tf/'
+                  }]
+              }]
+          ,   wg: "Web Applications Working Group"
+          ,   wgURI: "https://www.w3.org/2019/webapps/"
+          ,   wgPatentURI: "https://www.w3.org/2004/01/pp-impl/114929/status"
+          ,   edDraftURI:   "http://w3c.github.io/editing/docs/virtualKeyboard/"
+          , xref: ["WebIDL"
+          , "DOM"
+          , "HTML"
+          , "geometry"]
+        };
+    </script>
+    <script src='https://www.w3.org/Tools/respec/respec-w3c' class='remove'></script>
+</head>
+
+<body>
+    <section id='abstract'>
+        <p>
+            The {{VirtualKeyboard}} API provides web developers a more granular control of on-screen keyboard (virtual keyboard or VK). The API also provides information about VK geometry, such as rect occlusion and gives ability to opt-out from resizing of the visual viewport in the browser.
+        </p>
+    </section>
+    <section id='sotd'>
+      <p>
+      </p>
+    </section>
+    <section>
+        <h2>Introduction</h2>
+        <P>This section is non-normative.</P>
+        <p>
+            The Virtual Keyboard (VK) is the on-screen keyboard used for input in scenarios where a hardware keyboard may not be available. Unlike hardware keyboard, VK can adapt its shape to optimize for the expected type of input.
+            In addition, user agents respond to the presence of the VK, without any exposure of this information to web developers. This document proposes a new web API surface for VK that developers can use to enable better customization of their webpage's content and experiences.
+        </p>
+    </section>
+    <section id="conformance">
+        <p>
+          This specification defines conformance criteria that apply to a single
+          product: the <dfn id="dfn-user-agent">user agent</dfn> that implements the interfaces that
+          it contains.
+        </p>
+        <p>
+          Conformance requirements phrased as algorithms or specific steps may be
+          implemented in any manner, so long as the end result is equivalent. (In
+          particular, the algorithms defined in this specification are intended
+          to be easy to follow, and not intended to be performant.)
+        </p>
+      </section>
+    <section data-dfn-for="VirtualKeyboard">
+        <h2>The <dfn>VirtualKeyboard</dfn> Interface</h2>
+        <pre class="idl">
+            partial interface Navigator {
+                [SameObject] readonly attribute VirtualKeyboard virtualKeyboard;
+            };
+            
+            [Exposed=Window]
+            interface VirtualKeyboard : EventTarget {
+                undefined show();
```

Comment:
I think this should be `void` instead of `undefined`.

https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/virtualkeyboard/virtual_keyboard.idl;l=1?q=virtual_keyboard.idl&sq=&ss=chromium

Reply:
we were getting respec errors when we used `void`. Web IDL recommends using `undefined` instead of void https://heycam.github.io/webidl/#idl-undefined